pages/common/virt-qemu-run.md

```diff
@@ -0,0 +1,20 @@
+# virt-qemu-run
+
+> Experimental tool to run a QEMU Guest VM independent of libvirtd.
+> More information: <https://manned.org/virt-qemu-run>.
```

Comment:
```suggestion
> More information: <https://libvirt.org/manpages/virt-qemu-run.html>.
```
Since, author's documentation is available here. I would suggest linking to it.